ballerina/build.gradle

```diff
@@ -691,27 +691,17 @@ task testExamples() {
 
             // Disabling temporarily (https://github.com/ballerina-platform/ballerina-distribution/issues/1242)
             "ballerina-to-openapi",
-            "dynamic-listener",
             "xa-transactions",
             "retry-transactions",
             "non-transaction-retry",
             "local-transactions-with-handlers",
             "local-transactions",
             "log-api",
-            "async",
```

Comment:
Any reason why we remove these from tests?

Reply:
This is the ignore list. :smiley: Enabling disabled tests.